runtime/common/BaseRestRemoteClient.h

```diff
@@ -682,6 +682,20 @@ class BaseNvcfRuntimeClient : public cudaq::BaseRemoteRestRuntimeClient {
               const std::string &backendSimName, const std::string &kernelName,
               void (*kernelFunc)(void *), void *kernelArgs,
               std::uint64_t argsSize, std::string *optionalErrorMsg) override {
+    static const std::vector<std::string> DISALLOWED_EXECUTION_CONTEXT = {
```

Comment:
You could make this `constexpr` if you used `std::array<std::string_view, 1>` as the type.